scripts/buidler-hooks.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * These hooks are called by the Aragon Buidler plugin during the start task's lifecycle. Use them to perform custom tasks at certain entry points of the development build process, like deploying a token before a proxy is initialized, etc.
+ *
+ * Link them to the main buidler config file (buidler.config.js) in the `aragon.hooks` property.
+ *
+ * All hooks receive two parameters:
+ * 1) A params object that may contain other objects that pertain to the particular hook.
+ * 2) A "bre" or BuidlerRuntimeEnvironment object that contains enviroment objects like web3, Truffle artifacts, etc.
+ *
+ * Please see AragonConfigHooks, in the plugin's types for further details on these interfaces.
+ * https://github.com/aragon/buidler-aragon/blob/develop/src/types.ts#L22
```

Comment:
This line has now changed to [L31](https://github.com/aragon/buidler-aragon/blob/develop/src/types.ts#L31).

Do we export these types from the plugin? Might be nice to also say how a user could just import it themselves.